code/modules/mob/living/basic/slime/ai/controller.dm

```diff
@@ -19,6 +19,7 @@
 		/datum/ai_planning_subtree/basic_melee_attack_subtree/slime,
 		/datum/ai_planning_subtree/random_speech/slime,
 	)
+	can_idle = FALSE
```

Comment:
change interesting distance instead to be longer

Reply:
i worry about this since xenobio is isolated from the rest of the station in maps like metastation where most likely no one is going to be near it anyway meaning theyll always be idle